File: android/src/main/kotlin/uz/shs/video_player/models/TvProgram.kt
package uz.shs.video_player.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class TvProgram(
    @SerializedName("scheduledTime")
    val scheduledTime: String,
    @SerializedName("programTitle")
    val programTitle: String,
) : Serializable

File: lib/models/tv_program.dart
class TvProgram {
  const TvProgram({required this.scheduledTime, required this.programTitle});

  final String scheduledTime;
  final String programTitle;

  Map<String, dynamic> toMap() => {'programTitle': programTitle, 'scheduledTime': scheduledTime};

  @override
  String toString() => 'TvProgram{scheduledTime: $scheduledTime, programTitle: $programTitle}';
}
